### PR TITLE
Update collector and app version for v0.25.0

### PR DESCRIPTION
**Pull request by bryan-aguilar**
**Description:** Update collector version to v0.25.0. update app version to v0.11.0

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]

File: charts/adot-exporter-for-eks-on-ec2/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: adot-exporter-for-eks-on-ec2
 description: A Helm chart for collecting metrics using ADOT Collector to send to AWS monitoring services.
 type: application
-version: 0.10.0
-appVersion: "0.24.1"
+version: 0.11.0
+appVersion: "0.25.0"
 kubeVersion: ">=1.19.0-0"
```
Comment (rapphil) [inline charts/adot-exporter-for-eks-on-ec2/Chart.yaml:7]: Do we need to update this as well?

Comment (bryan-aguilar) [reply charts/adot-exporter-for-eks-on-ec2/Chart.yaml:7]: Yes, the CD is triggered by an increment to `version`. I believe `appversion` is a best practice to reflect the version of the application inside the chart and is the standard here.

Comment (rapphil) [reply charts/adot-exporter-for-eks-on-ec2/Chart.yaml:7]: I meant the `kubeVersion`.

Comment (bryan-aguilar) [reply charts/adot-exporter-for-eks-on-ec2/Chart.yaml:7]: Unclear if a new semver range should be applied. Created a new [issue](https://github.com/aws-observability/aws-otel-helm-charts/issues/84) to address this.

**Pull request merged**